### PR TITLE
PLATUI-1660: Adding documentation for external and internal users

### PR DESCRIPTION
**Pull request by JoPintoPaul**

### CODE REVIEW

File: docs/maintainers.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+     for `triage`
+   2. Read through the ticket, and see if it clearly falls into one of the following categories:
+      1. `accepted: bug` (if this is clearly a bug)
+         1. Create a bug ticket in Jira 
+         2. If the ticket seems like BAU, apply the `BAU` label to the **Jira ticket** 
+         3. Add a link in the Jira ticket to the **Github issue** (for closing once complete)
+         4. Apply the label `accepted: bug` to the **Github Issue** 
+         5. Reply to the original ticket with the [standard copy](https://github.com/hmrc/platui-support/blob/main/docs/template-responses.md)
+            for `accepted: bug` including the Jira ticket number 
+         6. Alert the @hobbit group in the Slack channel `#int-team-plat-ui`
```
Comment (ellamdav) [inline docs/maintainers.md]: looks like `@hobbit` has been renamed to `@platui-leads`

File: README.md [hunk 2]
```diff
@@ -1,4 +1,51 @@
...
+Please report any issues with this repo in Slack at `#team-plat-ui`.
+
+## Useful Links
+- [HMRC Design Patterns](https://design.tax.service.gov.uk/hmrc-design-patterns/) - documentation for the use of `hmrc-frontend` components
+- [hmrc-frontend](https://github.com/hmrc/hmrc-frontend/) - reusable Nunjucks HTML components for HMRC design patterns
+- [GOV.UK Design System](https://design-system.service.gov.uk/components/) - documentation for the use of `govuk-frontend` components
+- [govuk-frontend](https://github.com/alphagov/govuk-frontend/) - reusable Nunjucks HTML components from GOV.UK
+
+## Owning team documentation
+Instructions for team members maintaining this repository can be found [here](https://github.com/hmrc/platui-support/blob/main/docs/maintainers.md).
```
Comment (ellamdav) [inline README.md]: Can use relative links here, so local navigation works (eg. in intellij)
```suggestion
Instructions for team members maintaining this repository can be found [here](docs/maintainers.md).
```

File: docs/maintainers.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      2. `invalid`
+         1. Apply the label `invalid` 
+         2. Reply to the original ticket with the [standard copy](https://github.com/hmrc/platui-support/blob/main/docs/template-responses.md)
```
Comment (ellamdav) [inline docs/maintainers.md]: these can be relative links too

File: README.md (hunk 2 above)
Comment (JoPintoPaul) [reply README.md]: @ellamdav corrected, thank you.

File: docs/maintainers.md (hunk 1 above)
Comment (JoPintoPaul) [reply docs/maintainers.md]: @ellamdav corrected, thank you.

File: docs/maintainers.md (hunk 3 above)
Comment (JoPintoPaul) [reply docs/maintainers.md]: @ellamdav corrected, thank you.

**Review by ellamdav — APPROVED** [review]
LGTM 🚢

**Pull request merged**